core/scripts/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/smartcontractkit/chainlink/core/scripts
 
-go 1.21
+go 1.21.3
```

Comment:
/nit do we want to use 1.21.4? 

https://github.com/smartcontractkit/chainlink/blob/bd7e233d3a489e8dacae0a8c8539802b3ae10832/.tool-versions#L1

Reply:
I upgraded and this didn't change. I think it is propagated from a dependency as a minimal version perhaps? 🤷